pkg/function/kube_task.go

```diff
@@ -64,6 +65,18 @@ func kubeTask(ctx context.Context, cli kubernetes.Interface, namespace, image st
 		Command:      command,
 		PodOverride:  podOverride,
 	}
+	// Mark labels to pods with prefix `jobPrefix`. Add the jobID as reference to the origin for the pod.
```

Comment:
I don't see `jobPrefix` used. Is it `consts.LabelPrefix+"JobID"`?

Reply:
Yes, I corrected it.